metadata-models/src/main/pegasus/com/linkedin/dataset/ViewProperties.pdl

```diff
@@ -25,6 +25,13 @@ record ViewProperties {
    */
   viewLogic: string
 
+  /**
+   * The formatted view logic. This is particularly used for SQL sources, where the SQL
+   * logic is formatted for better readability, and with dbt, where this contains the
+   * compiled SQL logic.
+   */
+  formattedViewLogic: optional string
```

Comment:
Are we going to need corresponding frontend changes to read this property instead of `viewLogic`? I guess it should be `formattedViewLogic || viewLogic`

Reply:
yup the frontend changes are in https://github.com/datahub-project/datahub/pull/10028